target_chains/ethereum/contracts/forge-test/utils/WormholeTestUtils.t.sol

```diff
@@ -71,17 +126,113 @@ abstract contract WormholeTestUtils is Test {
 
         vaa = abi.encodePacked(
             uint8(1), // Version
-            uint32(0), // Guardian set index. it is initialized by 0
+            IWormhole(wormholeReceiverAddr).getCurrentGuardianSetIndex(), // Guardian set index. it is initialized by 0
             numSigners,
             signatures,
             body
         );
     }
+
+    function forgeVaa(
```

Comment:
Can we have a test with forgeItem="" to make sure it is correct without forging? like [this](https://github.com/pyth-network/pyth-crosschain/pull/908).